Gemfile

```diff
@@ -5,8 +5,9 @@ ruby '2.6.6'
 # Service/framework dependencies
 gem 'rails',
   git: 'http://github.com/rails/rails.git',
-  ref: '7101489293ece071013417aeed2da3aa1b0927c9'
+  ref: '207f116234374d6414593246240b05261a96fa3a'
```

Comment:
Sporadic bumping of Rails that I thought I'd do while breaking everything else at the same time 😉